api_tests/registrations/filters/test_filters.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+import pytest
+
+from osf.models import Node
+
+
+class RegistrationListFilteringMixin(object):
+
+    def setUp(self):
+        super(RegistrationListFilteringMixin, self).setUp()
+
+        self._setUp()
```

Comment:
The setup for the preprint_provider tests is different since all nodes must be the same provider, which is also why one of the tests is stubbed to pass.